client-src/elements/chromedash-roadmap-page.ts

```diff
@@ -60,24 +57,26 @@ export class ChromedashRoadmapPage extends LitElement {
   }
 
   handleResize() {
-    const containerWidth = this.sectionRef.value.offsetWidth;
+    const containerWidth = this.sectionRef.value!.offsetWidth;
     const margin = 16;
 
     let numColumns = 3;
-    if (containerWidth < 768) {
+    if (containerWidth && containerWidth < 768) {
       numColumns = 1;
-    } else if (containerWidth < 1152) {
+    } else if (containerWidth && containerWidth < 1152) {
```

Comment:
I think you can undo these conditions now that you're using `!` to initialize `containerWidth`.